stackset_template.tf

```diff
@@ -0,0 +1,88 @@
+locals {
+  stackset_template = <<TEMPLATE
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Parameters" : {
+        "ExpelCustomerOrganizationGUID" : {
+            "Type" : "String"
+        },
+         "ExpelAssumeRoleARN" : {
+            "Type" : "String"
+        }
+    },
+    "Resources": {
+        "IAMR7FYC": {
+            "Type": "AWS::IAM::Role",
+            "Properties": {
+                "ManagedPolicyArns": [
+                    {
+                        "Ref": "IAMMP12SQ7"
+                    }
+                ],
+                "RoleName": "ExpelRole",
+                "AssumeRolePolicyDocument": {
+                    "Version": "2012-10-17",
+                    "Statement": [
+                        {
+                            "Effect": "Allow",
+                            "Principal": {
+                                "AWS": { "Ref": "ExpelAssumeRoleARN" }
+                            },
+                            "Action": "sts:AssumeRole",
+                            "Condition": {
+                                "StringEquals": {
+                                    "sts:ExternalId": { "Ref": "ExpelCustomerOrganizationGUID" }
+                                }
+                            }
+                        }
+                    ]
+                }
+            }
+        },
+        "IAMMP12SQ7": {
+            "Type": "AWS::IAM::ManagedPolicy",
+            "Properties": {
+                "ManagedPolicyName": "ExpelAccessPolicy",
+                "PolicyDocument": {
+                    "Version": "2012-10-17",
+                    "Statement": [
+                        {
+                            "Effect": "Allow",
+                            "Action": [
+                                "cloudtrail:DescribeTrails",
```

Comment:
Is it possible to add a comment in here that this is a duplicate of the other one?

If it's not possible in here because this is a here-doc, can you put the comment at the top of the file?